src/components/Form/Form.styled.jsx

```diff
@@ -38,14 +37,31 @@ export const FormButtonPrimary = styled.button`
     box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   }
 
-  @media (min-width: 1025px) {
-    width: 200px;
+  @media (min-width: 1200px) {
+    width: 230px;
   }
+
+  @media (max-width: 1025px) {
+    width: 230px;
+  }
+
+  @media (max-width: 768px) {
+    width: 292px;
+  }
+
+  @media (max-width: 480px) {
+    width: 293px;
+  }
+
+  @media (max-width: 280px) {
+    width: 203px;
+  }
```

Comment:
doesn't look good in 1056px screen size
![imagen](https://user-images.githubusercontent.com/66505715/157891900-f27d19d6-ccf2-4090-8915-8d08d8899b28.png)

doesn't look good in 376px screen size
![imagen](https://user-images.githubusercontent.com/66505715/157892080-bf62ec40-76c1-44ef-8a88-a5aeeb4069f3.png)

Reply:
I propose this solution using flex, it's just a proposal since I don't know if this can cause problems in another view and obvious can exists a better solution

1. the form currently it has `width: 80%`, you can move it to `100%`
2. the button have a static px value in width, since the button container has flex, you can change it for `width: 100%` in each button, flex will do its thing dividing the space between the two buttons
3. the inputs now, since you have incremented the form size from 80% to 100%, will increase his size. This is because inputs have `width: 100%`, please move this width to `width: 80%` and doesn't should exist difference

Now the buttons look good on the phone, when is 1 button it is occupying 100% of the width same as design

![imagen](https://user-images.githubusercontent.com/66505715/157890149-0a4b6e9e-54d4-4b01-b49c-72e1eeacb2b2.png)

And when are 2 buttons, they're occupying 50% and 50% of form size, which is bigger of  inputs, then looks similar to:

![imagen](https://user-images.githubusercontent.com/66505715/157890280-03357480-f309-47c0-a39f-2080fd382285.png)

For desktop:

1. Add a media query where desktop size start
2. Put in this media query that button should have a `max-width: 230px` or maybe `max-width: 273px` ?

Then the form should look similar to the design when is just 1 button or are 2 buttons:
![imagen](https://user-images.githubusercontent.com/66505715/157890746-5ce32cf8-dc54-4444-99c4-de04a4a95170.png)